core/engine/src/vm/call_frame/mod.rs

```diff
@@ -78,6 +79,19 @@ impl CallFrame {
     pub const fn code_block(&self) -> &Gc<CodeBlock> {
         &self.code_block
     }
+
+    /// Retrieves a tuple of `(`[`JsString`]`, `[`SourcePath`]`, `[`Position`]`)` to know the
+    /// location of the call frame.
+    #[inline]
+    #[must_use]
+    pub fn position(&self) -> (JsString, SourcePath, Option<Position>) {
```

Comment:
Consider returning a struct with the fields

```
using boa_ast::Position as AstPosition;

struct SourcePosition {
    function_name: JsString,
    path: SourcePath,
    position: AstPosition,
}
```

Reply:
I was too lazy lol.